cli/crates/common/src/consts.rs

```diff
@@ -23,7 +23,7 @@ pub const AUTHORIZERS_DIRECTORY_NAME: &str = "auth";
 /// the bun installation directory within ~/.grafbase
 pub const BUN_DIRECTORY_NAME: &str = "bun";
 /// the tracing filter to be used when tracing is on
-pub const TRACE_LOG_FILTER: &str = "grafbase=trace,grafbase_local_common=trace,grafbase_local_server=trace,grafbase_local_backend=trace,tower_http=debug,federated_dev=trace,postgres-connector-types=trace,engine_v2=debug";
```

Comment:
Oh, I also removed `tower_http` here because it's noisy and I'm not sure who gives a shit.  If you _do_ give a shit please feel free to call me names and I'll revert.